app/views/content/become-a-further-education-teacher.md

```diff
@@ -16,21 +17,27 @@ keywords:
   - NVQ
   - BTEC
   - HND
-  - apprencticeship
 ---
 
-Further education is any study that takes place after secondary school but outside of higher education (e.g. university).
 
-In further education, teaching doesn’t always happen in a classroom. It can take place in community centres, workplaces and private training centres. 
+If you’re interested in teaching young people and adults over 16 who are not studying for a degree, then further education might be for you. 
 
-You could teach a diverse set of learners, including young people and adults.
+## What is further education?
 
-You do not need a degree to start teaching in further education. 
+It covers three key areas: 
 
-## Training to teach in further education
+* vocational courses including plumbing, manufacturing and health care
+* academic classroom-based learning of subjects including English and maths, including A levels in a sixth form college
+* basic skills courses on literacy, numeracy, English for speakers of other languages, and special needs
 
-To find out more, visit the [teach in further education website](https://www.teach-in-further-education.campaign.gov.uk/).
+Whichever route you choose, you could teach a diverse set of learners, including young people and adults.
```

Comment:
"including young people and adults" feels a bit weird at the end of this sentence - who else is there? And we've said 'young people and adults' earlier on so wondering if this could just be removed.

(NB this is very minor and wouldn't stop this going live @philspray!)